cmd/provisioner-localpv/app/helper_hostpath.go

```diff
@@ -140,14 +162,15 @@ func (p *Provisioner) createCleanupPod(pOpts *HelperPodOptions) error {
 		return vErr
 	}
 
-	cPod, err := p.launchPod(config)
+	_, err := p.launchPod(config)
 	if err != nil {
 		return err
 	}
 
-	if err := p.exitPod(cPod); err != nil {
+	klog.Infof("Will not delete the helperPOD......")
+	/*if err := p.exitPod(cPod); err != nil {
```

Comment:
`exitPod()` should be executed so that the pods  cleanup pods are removed after completing the cleanup

Reply:
Done